spec/features/admin/admin_modifies_items_spec.rb

```diff
@@ -25,7 +25,7 @@
     click_on "Edit Items"
   end
 
-  xscenario 'can view item index' do
+  scenario 'can view item index' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.